.flake8

```diff
@@ -41,6 +41,7 @@ extend-ignore =
   WPS300,  # "Found local folder import" -- nothing bad about this
   # An opposite consistency expectation is currently enforced
   # by pylint via: useless-object-inheritance (R0205):
+  WPS305,  # "Found `f` string", allow the use of f strings
```

Comment:
This is the wrong line to add this. Just look at the comment above that refers to the line below (WPS306) by having a colon at the end of the comment.

Move this addition to line 42. Plus maybe add motivation — as in "we really want to use f-strings and it's legit". I'd recommend extending the comment by adding the explanation after `--` like it's done on line 41 to prevent future similar separation of comments from the referred content.

Reply:
Could also refer to https://github.com/ansible/ansible-navigator/discussions/717#discussioncomment-1791048 in the comment.

Reply:
thx, updated. Many reasons to use them. I find readability the biggest win

Reply:
resolving as approved after change was implemented